scheduler/src/cook/config.clj

```diff
@@ -483,7 +483,11 @@
                              :pod-condition-unschedulable-seconds 60
                              :reconnect-delay-ms 60000
                              :set-container-cpu-limit? true}
-                            kubernetes)))}))
+                            kubernetes)))
+     :matching-settings (fnk [[:config {matching-settings {}}]]
```

Comment:
Maybe just `:matching` since we're already under `:settings`?

Reply:
seems a bit too short. but I don't have a strong preference

Reply:
What's the downside to short?

Reply:
how about offer-matching?